src/components/AddItem.js

```diff
@@ -24,8 +26,24 @@ const AddItem = ({ token }) => {
         });
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
+    const itemNameInput = formData.itemName;
+
+    // check to see if the itemName already exists in the user's list - convert input to lowercase and remove any punctuation & white space
+    const cleanInput = itemNameInput.toLowerCase().replace(/[^\w\s]|/g, '');
+
```

Comment:
This may be a holdover from @MikeBLambert 's previous comment and you may have already addressed this, but just in case, when I viewed the application with Netlify and entered similar items with slight changes, I was able to add them to the list. Just wanted to pop this in here in case it's a tenacious bug.  
<img width="113" alt="snapshot of list" src="https://user-images.githubusercontent.com/36529777/105210286-b5e86a00-5aff-11eb-8272-56ca6552344c.PNG">

Reply:
I think @rachelrohrbach and I fixed this issue. We also refactored the regex.